docs/common-coreboot-docs/checkbox.md

```diff
@@ -0,0 +1,222 @@
+following command:
+
+```bash
+sudo visudo
+```
+
+Append the following entry at the end of file to run ALL command without a
+password for a user:
+
+```bash
+user ALL=(ALL) NOPASSWD:ALL
+```
+
+To set up automatic login so that a password is not required during testing
+reboot, poweroff etc.:
+
+1. Press the `SUPER_KEY`, type in `Users` and click Enter. Window with options to
+change should appear.
+1. Next click the user name you want to enable and disable automatic login.
+1. Then on the top right side of the window click `Unlock...` button.
+1. You will be required to enter your user password to continue.
+1. You should now be able to toggle Automatic Login button to enable.
+1. Restart your system for changes to take effect.
+
+## Run automatic validation
+
+1. Start checkbox. To do this, open terminal and run the following command:
+
+    ```bash
+    $ sudo checkbox-cli
+    ```
+
+    After using the above-mentioned command, the following menu should appear:
+
+    ```bash
+    ┌──────────────────────────────────────────────────────────────────────────────┐
+    │    (X) (Deprecated) Fully Automatic Client Certification Tests               │
+    │    ( ) After suspend Dock Cert blocker tests                                 │
+    │    ( ) After suspend Dock Cert tests                                         │
+    │    ( ) After suspend Dock Hot-plug Cert tests                                │
+    │    ( ) After suspend Dock Hot-plug tests                                     │
+    │    ( ) After suspend LED and oops tests (using special sleep key)            │
+    │    ( ) After suspend LED and oops tests (using special sleep key, cert.      │
+    │        blockers only)                                                        │
+    │    ( ) After suspend automated USB 3 write/read/compare tests on storage     │
+    │        devices                                                               │
+    │    ( ) After suspend automated USB write/read/compare tests on storage       │
+    │        devices                                                               │
+    │    ( ) After suspend reference tests                                         │
+    │    ( ) After suspend reference tests (automated)                             │
+    │    ( ) After suspend reference tests (certification blockers only)           │
+    │    ( ) After suspend tests (discrete GPU automated)                          │
+    │    ( ) After suspend tests (discrete GPU manual)                             │
+    │    ( ) After suspend tests (discrete GPU)                                    │
+    │    ( ) After suspend tests (discrete GPU, certification blockers only)       |
+            .
+            .
+            . etc.
+    └──────────────────────────────────────────────────────────────────────────────┘
+    ```
+
+1. Select test suites (`SPACE` - select, `ARROWS` - navigation, `Enter` -
```

Comment:
We should add here that marking the first option starts the full check.

Reply:
added